ui/app/styles/core/helpers.scss

```diff
@@ -12,6 +12,9 @@
 .is-no-underline {
   text-decoration: none;
 }
+.has-line-height-1 {
```

Comment:
Want to make sure @Monkeychip is aware of these additions for the upcoming css organization!